From 10b43f3d9fe1ee0cccf3c80e603e6cce6dedd333 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 25 Oct 2010 18:35:19 +0000
Subject: [PATCH 0050/1077] rules added to common makefile: lint, dep,
 cleandep, clean-lint, common-clean-lint, globaldep, common-globaldep,
 finalize-globaldep

git-svn-id: svn://localhost/SelfServiceCommon/trunk@54 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Build/cpp/makefile.mak       | 18 +++++++++++++++++-
 .../Massai/cpp/BillingService/makefile.mak     |  9 +++++++++
 .../cpp/BillingService/static/makefile.mak     |  3 +++
 .../Massai/cpp/CfgClnt/dll/makefile.mak        |  3 +++
 .../Massai/cpp/CfgClnt/static/makefile.mak     |  3 +++
 .../Massai/cpp/CfgTools/makefile.mak           | 18 ++++++++++++++++++
 .../Massai/cpp/CfgTools/static/makefile.mak    | 18 ++++++++++++++++++
 .../Massai/cpp/CfgXMgrB/makefile.mak           |  8 ++++++++
 .../Massai/cpp/CfgXMgrB/static/makefile.mak    |  7 +++++++
 .../Massai/cpp/Controller/makefile.mak         |  8 ++++++++
 .../Massai/cpp/LogService/makefile.mak         | 14 ++++++++++++++
 .../Massai/cpp/MailService/makefile.mak        |  9 +++++++++
 .../Massai/cpp/SecureLogging/makefile.mak      |  6 ++++++
 .../Massai/cpp/StatisticsService/makefile.mak  |  6 ++++++
 .../Services/cpp/Statistics/makefile.mak       |  3 +++
 .../cpp/Statistics/static/makefile.mak         |  3 +++
 16 files changed, 135 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Build/cpp/makefile.mak b/SelfServiceCommon/Build/cpp/makefile.mak
index 6fb9b1cc..6bea9712 100644
--- a/SelfServiceCommon/Build/cpp/makefile.mak
+++ b/SelfServiceCommon/Build/cpp/makefile.mak
@@ -14,7 +14,23 @@ clean:: \
 
 very-clean: \
     clean
-    
+
+lint: \
+    common-lint
+
+dep: \
+    common-dep
+
+cleandep: \
+    common-cleandep
+
+clean-lint: \
+    common-clean-lint
+
+globaldep: \
+    common-globaldep \
+    finalize-globaldep
+
 java:
     cd $(BUILDROOT)\Build\java && $(ANT) all-common
 
diff --git a/SelfServiceCommon/Massai/cpp/BillingService/makefile.mak b/SelfServiceCommon/Massai/cpp/BillingService/makefile.mak
index 2a3c40f1..35e592ca 100644
--- a/SelfServiceCommon/Massai/cpp/BillingService/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/BillingService/makefile.mak
@@ -22,6 +22,9 @@ MY_LIB1_OBJS = \
     $(_OBJ)\BillingService_skel.obj \
     $(_OBJ)\billing.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN1 = \
@@ -51,6 +54,9 @@ MY_BIN1_USER_LIBS = \
 MY_BIN1_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -73,6 +79,9 @@ MY_BIN2_USER_LIBS = \
 MY_BIN2_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/BillingService/static/makefile.mak b/SelfServiceCommon/Massai/cpp/BillingService/static/makefile.mak
index 3e70bc1b..a1333bd3 100644
--- a/SelfServiceCommon/Massai/cpp/BillingService/static/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/BillingService/static/makefile.mak
@@ -27,6 +27,9 @@ MY_LIB1_OBJS = \
     $(_OBJ)\BillingService_skel.obj \
     $(_OBJ)\billing.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/CfgClnt/dll/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgClnt/dll/makefile.mak
index 9790dbbc..b1175b5b 100644
--- a/SelfServiceCommon/Massai/cpp/CfgClnt/dll/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgClnt/dll/makefile.mak
@@ -49,6 +49,9 @@ MY_LIB1_OBJS = \
 MY_LIB1_USER_LIBS= \
     $(LIB_MSYSTOOLS)
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN1 = \
diff --git a/SelfServiceCommon/Massai/cpp/CfgClnt/static/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgClnt/static/makefile.mak
index 49c8db01..0c96c62c 100644
--- a/SelfServiceCommon/Massai/cpp/CfgClnt/static/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgClnt/static/makefile.mak
@@ -42,6 +42,9 @@ MY_LIB1_OBJS = \
 MY_LIB1_USER_LIBS= \
     $(LIB_MSYSTOOLS)
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_DLL1 = \
diff --git a/SelfServiceCommon/Massai/cpp/CfgTools/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgTools/makefile.mak
index 384813cb..af6df0e2 100644
--- a/SelfServiceCommon/Massai/cpp/CfgTools/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgTools/makefile.mak
@@ -37,6 +37,9 @@ MY_BIN1_USER_LIBS= \
 MY_BIN1_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -58,6 +61,9 @@ MY_BIN2_USER_LIBS = \
 MY_BIN2_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN3 = \
@@ -79,6 +85,9 @@ MY_BIN3_USER_LIBS = \
 MY_BIN3_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN3_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN4 = \
@@ -100,6 +109,9 @@ MY_BIN4_USER_LIBS = \
 MY_BIN4_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN4_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN5 = \
@@ -121,6 +133,9 @@ MY_BIN5_USER_LIBS = \
 MY_BIN5_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN5_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN6 = \
@@ -142,6 +157,9 @@ MY_BIN6_USER_LIBS = \
 MY_BIN6_LINKLIBS = \
     $(SYS_ACETAO)
 
+MY_BIN6_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/CfgTools/static/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgTools/static/makefile.mak
index 3f18cfdf..641701dc 100644
--- a/SelfServiceCommon/Massai/cpp/CfgTools/static/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgTools/static/makefile.mak
@@ -43,6 +43,9 @@ MY_BIN1_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -65,6 +68,9 @@ MY_BIN2_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN3 = \
@@ -87,6 +93,9 @@ MY_BIN3_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN3_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN4 = \
@@ -109,6 +118,9 @@ MY_BIN4_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN4_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN5 = \
@@ -131,6 +143,9 @@ MY_BIN5_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN5_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN6 = \
@@ -153,6 +168,9 @@ MY_BIN6_LINKLIBS = \
     $(SYS_ACETAO_S) \
     $(SYS_ADVAPI32)
 
+MY_BIN6_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/CfgXMgrB/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgXMgrB/makefile.mak
index bd928514..25abf491 100644
--- a/SelfServiceCommon/Massai/cpp/CfgXMgrB/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgXMgrB/makefile.mak
@@ -74,6 +74,9 @@ MY_BIN1_LINKLIBS = \
 MY_BIN1_RES = \
     $(_OBJ)\CfgXMgrB.res
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -98,4 +101,9 @@ MY_BIN2_RES = \
 MY_BIN2_LDLIBS = \
     -subsystem:console,4.0
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
+######################################################################
+
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/CfgXMgrB/static/makefile.mak b/SelfServiceCommon/Massai/cpp/CfgXMgrB/static/makefile.mak
index 431e657c..b46d9aa6 100644
--- a/SelfServiceCommon/Massai/cpp/CfgXMgrB/static/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/CfgXMgrB/static/makefile.mak
@@ -83,6 +83,8 @@ MY_BIN1_LINKLIBS = \
 MY_BIN1_RES = \
     $(_OBJ)\CfgXMgrB.res
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
 
 ######################################################################
 
@@ -108,4 +110,9 @@ MY_BIN2_RES = \
 MY_BIN2_LDLIBS = \
     -subsystem:console,4.0
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
+######################################################################
+
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/Controller/makefile.mak b/SelfServiceCommon/Massai/cpp/Controller/makefile.mak
index f8dd11dc..9092f519 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/Controller/makefile.mak
@@ -61,6 +61,9 @@ MY_BIN1_LINKLIBS = \
     $(STDLINKLIBS) \
     $(SYS_COMCTL32)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -102,4 +105,9 @@ MY_BIN3_LINKLIBS = \
 MY_BIN3_LDFLAGS = \
     -subsystem:console
 
+MY_BIN3_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
+######################################################################
+
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/LogService/makefile.mak b/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
index 596985f9..71add37a 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/LogService/makefile.mak
@@ -70,6 +70,8 @@ MY_DLL1_LINKLIBS = \
 MY_DLL1_RES = \
     $(_OBJ)\alerts.res
 
+MY_DLL1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
 
 ######################################################################
 
@@ -114,6 +116,9 @@ MY_DLL2_LINKLIBS = \
     $(STDLINKLIBS) \
     $(SYS_SNMPAPI)
 
+MY_DLL2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN1 = \
@@ -150,6 +155,9 @@ MY_BIN1_USER_LIBS = \
 MY_BIN1_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -169,6 +177,9 @@ MY_BIN2_USER_LIBS = \
 MY_BIN2_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN3 = \
@@ -215,6 +226,9 @@ MY_BIN4_LINKLIBS = \
 MY_BIN4_RES = \
     $(_OBJ)\logflush.res
 
+MY_BIN4_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN5 = \
diff --git a/SelfServiceCommon/Massai/cpp/MailService/makefile.mak b/SelfServiceCommon/Massai/cpp/MailService/makefile.mak
index 6d4ef903..261dad04 100644
--- a/SelfServiceCommon/Massai/cpp/MailService/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/MailService/makefile.mak
@@ -33,6 +33,9 @@ MY_LIB1_OBJS = \
     $(_OBJ)\MailService_skel.obj \
     $(_OBJ)\mailer.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN1 = \
@@ -57,6 +60,9 @@ MY_BIN1_USER_LIBS = \
 MY_BIN1_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -76,6 +82,9 @@ MY_BIN2_USER_LIBS = \
 MY_BIN2_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN3 = \
diff --git a/SelfServiceCommon/Massai/cpp/SecureLogging/makefile.mak b/SelfServiceCommon/Massai/cpp/SecureLogging/makefile.mak
index dea57c34..67da9818 100644
--- a/SelfServiceCommon/Massai/cpp/SecureLogging/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/SecureLogging/makefile.mak
@@ -45,6 +45,9 @@ MY_BIN1_USER_LIBS = \
 MY_BIN1_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -73,6 +76,9 @@ MY_LIB1_OBJS = \
     $(_OBJ)\SecureLoggingService.obj \
     $(_OBJ)\SecureLoggingService_skel.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Massai/cpp/StatisticsService/makefile.mak b/SelfServiceCommon/Massai/cpp/StatisticsService/makefile.mak
index ed5f96a6..819c334a 100644
--- a/SelfServiceCommon/Massai/cpp/StatisticsService/makefile.mak
+++ b/SelfServiceCommon/Massai/cpp/StatisticsService/makefile.mak
@@ -47,6 +47,9 @@ MY_BIN1_USER_LIBS = \
 MY_BIN1_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 MY_BIN2 = \
@@ -68,6 +71,9 @@ MY_BIN2_USER_LIBS = \
 MY_BIN2_LINKLIBS = \
     $(STDLINKLIBS)
 
+MY_BIN2_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 ######################################################################
 
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Services/cpp/Statistics/makefile.mak b/SelfServiceCommon/Services/cpp/Statistics/makefile.mak
index 833312f4..5ba4a3db 100644
--- a/SelfServiceCommon/Services/cpp/Statistics/makefile.mak
+++ b/SelfServiceCommon/Services/cpp/Statistics/makefile.mak
@@ -19,4 +19,7 @@ MY_LIB1_OBJS = \
     $(_OBJ)\StatisticsService.obj \
     $(_OBJ)\StatisticsService_skel.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
diff --git a/SelfServiceCommon/Services/cpp/Statistics/static/makefile.mak b/SelfServiceCommon/Services/cpp/Statistics/static/makefile.mak
index 1d3fa081..5db6259e 100644
--- a/SelfServiceCommon/Services/cpp/Statistics/static/makefile.mak
+++ b/SelfServiceCommon/Services/cpp/Statistics/static/makefile.mak
@@ -24,4 +24,7 @@ MY_LIB1_OBJS = \
     $(_OBJ)\StatisticsService.obj \
     $(_OBJ)\StatisticsService_skel.obj
 
+MY_LIB1_GEN_DIRS = \
+    $(GEN_MASSAI_COMMON)
+
 !include $(BUILDROOT)\Build\cpp\makefile.rules.mak
-- 
2.41.0.windows.1

